README.md

```diff
@@ -31,18 +28,28 @@ plugins {
 }
 ```
 
-This is all that is needed. Run your pipeline with the usual command.
+This is all that is needed. Run your pipeline with the usual command:
+```bash
+nextflow run <pipeline_name>.nf 
+```
 
-You can find more information on plugins in the [Nextflow documentation](https://www.nextflow.io/docs/latest/plugins.html#plugins).
+More details are available in the Nextflow [plugin documentation](https://www.nextflow.io/docs/latest/plugins.html#plugins) and the [configuration guide](https://www.nextflow.io/docs/latest/config.html). 
 
-> [!NOTE]
-> To test the plugin prior to its first release, refer to the [contributing documentation](contributing/setup.md).
+> **⚠️ Note**  
+> The usage of GPUs is not yet supported.
```

Comment:
According to the [docs](https://python-markdown.github.io/extensions/admonition/), you should this would be the correct admonition syntax in your case
```suggestion
!!! note
  The usage of GPUs is not yet supported.
```